.travis.yml

```diff
@@ -1,24 +1,22 @@
-sudo: required
-
 language: bash
+services: docker
 
-services:
-  - docker
-
-install:
-  - docker build -t quay.io/azavea/django:${TRAVIS_COMMIT:0:7} .
+env:
+  - VERSION=1.9 PYTHON_VERSION=2.7 PG_MAJOR=9.5 PG_VERSION=9.4+165+deb8u2 VARIANT=slim
+  - VERSION=1.9 PYTHON_VERSION=3.5 PG_MAJOR=9.5 PG_VERSION=9.4+165+deb8u2 VARIANT=slim
+  - VERSION=1.9 PYTHON_VERSION=2.7 PG_MAJOR=9.5 PG_VERSION=9.5.4-r0 VARIANT=alpine
+  - VERSION=1.9 PYTHON_VERSION=3.5 PG_MAJOR=9.5 PG_VERSION=9.5.4-r0 VARIANT=alpine
+  - VERSION=1.10 PYTHON_VERSION=2.7 PG_MAJOR=9.6 PG_VERSION=9.6+180.pgdg80+1 VARIANT=slim
+  - VERSION=1.10 PYTHON_VERSION=3.5 PG_MAJOR=9.6 PG_VERSION=9.6+180.pgdg80+1 VARIANT=slim
+  - VERSION=1.10 PYTHON_VERSION=2.7 PG_MAJOR=9.6 PG_VERSION=9.5.4-r0 VARIANT=alpine
+  - VERSION=1.10 PYTHON_VERSION=3.5 PG_MAJOR=9.6 PG_VERSION=9.5.4-r0 VARIANT=alpine
```

Comment:
Adding support for the current Python release would be good, which is 3.6: https://www.python.org/downloads/release/python-360/